lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +80,27 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  }
+  return mconfig;
+}
+
 int main() {
   signalMask();
 
-  magma::init_logging("sctpd");
-  magma::set_verbosity(MDEBUG);
+  auto mconfig = load_mconfig();
+
+  magma::init_logging(SCTPD_SERVICE);
+  magma::set_verbosity(mconfig.log_level());
```

Comment:
It seems like the log levels between mconfig and verbosity are inverted so INFO is interpreted as ERROR (not just here but in other services too). Is this a known issue or am I misunderstanding something?

Reply:
That's interesting... How do other services handle it? I remember we have some logic to remap log values to glog log levels.

Reply:
I think there's a mismatch between the values in `magma::orc8r::LogLevel` and the `magma_logging.h` constants. Putting this into the main
```cpp
magma::set_verbosity(magma::orc8r::LogLevel::DEBUG);
magma::set_verbosity(magma::orc8r::LogLevel::INFO);
magma::set_verbosity(magma::orc8r::LogLevel::WARNING);
magma::set_verbosity(magma::orc8r::LogLevel::ERROR);
magma::set_verbosity(magma::orc8r::LogLevel::FATAL);
magma::set_verbosity(MDEBUG);
magma::set_verbosity(MINFO);
magma::set_verbosity(MWARNING);
magma::set_verbosity(MERROR);
magma::set_verbosity(MFATAL);
```
results in:
```
I1216 15:31:06.167461 203384 magma_logging_init.h:28] Setting verbosity to 0
I1216 15:31:06.167547 203384 magma_logging_init.h:28] Setting verbosity to 1
I1216 15:31:06.167569 203384 magma_logging_init.h:28] Setting verbosity to 2
I1216 15:31:06.167587 203384 magma_logging_init.h:28] Setting verbosity to 3
I1216 15:31:06.167603 203384 magma_logging_init.h:28] Setting verbosity to 4
I1216 15:31:06.167619 203384 magma_logging_init.h:28] Setting verbosity to 4
I1216 15:31:06.167635 203384 magma_logging_init.h:28] Setting verbosity to 3
I1216 15:31:06.167675 203384 magma_logging_init.h:28] Setting verbosity to 2
I1216 15:31:06.167696 203384 magma_logging_init.h:28] Setting verbosity to 1
I1216 15:31:06.167714 203384 magma_logging_init.h:28] Setting verbosity to 0
```
So there's still a conversion missing. Sessiond, liagent and connection tracker have a function  `get_log_verbosity` that will also return the mconfig value directly if it's not overwritten by the config.

Reply:
yikes - we should probably fix that :D Can you file an issue?

Reply:
#10911